api/actions/createWorkItem.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
     fn: async function (req) {
         console.log('createWorkItem Action called with req: ', req);
         const {
-            issue: { labels, url, body, milestone }
+            issue: { labels, html_url, body, milestone }
```

Comment:
switching the url we use to html_url so it is clickable on a browser. url gives the API url.